<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spacer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Images/search_icon.png</normaloff>:/Images/search_icon.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Поиск АЗС</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="SearchForm" name="widgetSearch" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ListNetworkForm" name="widgetListNetworks" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QToolButton" name="toolButtonGoHome">
      <property name="text">
       <string>На главную</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/Images/homepage_icon.png</normaloff>:/Images/homepage_icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSprav">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="actionNetworkAzs"/>
    <addaction name="actionUsers"/>
   </widget>
   <addaction name="menuSprav"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNetworkAzs"/>
   <addaction name="actionUsers"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Images/setting_icon.png</normaloff>:/Images/setting_icon.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionNetworkAzs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Images/gas station_icon.png</normaloff>:/Images/gas station_icon.png</iconset>
   </property>
   <property name="text">
    <string>Сети АЗС</string>
   </property>
  </action>
  <action name="actionUsers">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Images/users_icon.png</normaloff>:/Images/users_icon.png</iconset>
   </property>
   <property name="text">
    <string>Пользователи</string>
   </property>
   <property name="toolTip">
    <string>Справочник пользователей</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchForm</class>
   <extends>QWidget</extends>
   <header>Forms\searchform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListNetworkForm</class>
   <extends>QWidget</extends>
   <header>Forms/listnetworkform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
